frontend/public/components/storage/create-pvc.tsx

```diff
@@ -150,6 +152,8 @@ export class CreatePVCForm extends React.Component<CreatePVCFormProps, CreatePVC
     return obj;
   };
 
+  onlyPvcSCs = (sc: StorageClass) => !filterScOnProvisioner(sc, 'noobaa.io/obc');
```

Comment:
See previous comment about inverting the return value of your filter function. Just make the filter function return the correct value and don't invert it:
```suggestion
  onlyPvcSCs = (sc: StorageClass) => filterScOnProvisioner(sc, 'noobaa.io/obc');
```